components/layout/NavBar.tsx

```diff
@@ -19,11 +21,38 @@ const navMenu: navText[] = [
 ];
 
 const NavBar: React.FC = () => {
+  const matches = useMediaQuery("(min-width: 768px)");
+  const { scrollY } = useScroll();
   const [toggleMenu, setToggleMenu] = useState(false);
+  const [isTransparent, setIsTransparent] = useState(true);
+
+  useMotionValueEvent(scrollY, "change", (latest) => {
+    if (latest > 7) {
+      setIsTransparent(false);
+    } else {
+      setIsTransparent(true);
+    }
+  });
 
   return (
-    <section className="w-full max-w-screen-2xl mx-auto flex justify-center items-center fixed z-50">
-      <div className="w-11/12 h-[8rem] flex justify-between items-center">
+    <motion.section
```

Comment:
- please make the floating nav thinner and further up on the viewport.

![image](https://github.com/user-attachments/assets/37a40f88-8571-4724-a372-5e101ec50330)